test/Libraries/DynamoPythonTests/PythonEditTests.cs

```diff
@@ -251,28 +385,42 @@ public void ReturnIronPythonDictionary_AsDynamoDictionary()
 
             var guid = "490a8d54d0fa4782ae18c81f6eef8306";
 
-            AssertPreviewValue(guid, new Dictionary<string, int> {{"abc", 123}, {"def", 10}});
-        }
+            var nodeModel = ViewModel.Model.CurrentWorkspace.NodeFromWorkspace(guid);
+            var pynode = nodeModel as PythonNode;
 
-        private void UpdatePythonNodeContent(ModelBase pythonNode, string value)
-        {
-            var command = new DynCmd.UpdateModelValueCommand(
-                System.Guid.Empty, pythonNode.GUID, "ScriptContent", value);
+            foreach (var pythonEngine in PythonNodeBase.pythonEngineVersionsList)
+            {
+                UpdateEnginePropertyForPythonNode(pynode, pythonEngine);
 
-            ViewModel.ExecuteCommand(command);
+                ViewModel.HomeSpace.Run();
+
+                AssertPreviewValue(guid, new Dictionary<string, int> { { "abc", 123 }, { "def", 10 } });
+            }
         }
 
         [Test]
+        // This is failing for CPython3 Engine. 
```

Comment:
Should we mark this as failing and track as a follow-up?

Reply:
Marked it as failure.